src/box/lua/tuple.c

```diff
@@ -649,7 +649,7 @@ lbox_tuple_transform(struct lua_State *L)
 static int
 lbox_tuple_field_by_path(struct lua_State *L)
 {
-	struct tuple *tuple = luaT_istuple(L, 1);
+	struct tuple *tuple = luaT_checktuple(L, 1);
```

Comment:
Nit: The assertion for `tuple` is no longer needed. Let's remove.

Let's also remove the assertion for `path` and instead write:

```C
const char *path = luaL_checklstring(L, 2, &len);
```

Reply:
Fixed.